.github/workflows/release.yml

```diff
@@ -12,6 +12,10 @@ jobs:
   release:
     name: Release
     runs-on: ubuntu-latest
+    permissions:
+      actions: write
+      deployments: write
```

Comment:
Looking at the changeset actions script, I'm not sure if deployments is actually needed. 

https://github.com/changesets/action/blob/main/scripts/release.js

I also wonder if the script ultimately needs `contents` perms  to create the tags

https://docs.github.com/en/enterprise-cloud@latest/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents

Reply:
I see that they also use changesets. It looks like they still have successful releases, so perhaps `contents` and `pull-requests` is enough to unblock. I updated the permission fields.